federation/pkg/federation-controller/replicaset/replicasetcontroller.go

```diff
@@ -142,7 +142,7 @@ func NewReplicaSetController(federationClient fedclientset.Interface) *ReplicaSe
 			&extensionsv1.ReplicaSet{},
 			controller.NoResyncPeriodFunc(),
 			fedutil.NewTriggerOnAllChanges(
-				func(obj runtime.Object) { frsc.deliverLocalReplicaSet(obj, replicaSetReviewDelay) },
+				func(obj runtime.Object) { frsc.deliverReplicaSetObj(obj, replicaSetReviewDelay) },
```

Comment:
Looking at the code, there are behavioral changes between the two. The new funs does not list the replicasets.
Is that intentional?